Automattic/wp-calypso
# Create a environment badge component

Adress feedback from https://github.com/Automattic/wp-calypso/pull/22175#discussion_r167022583. [Earlier approach](https://github.com/Automattic/wp-calypso/pull/8459/files#diff-8238a46b17116c7bc31a42e5a32ffea8).

Create an `<EnvironmentBadge />` component out of markup which is currently used in templates on the server to display this little badge on the bottom right corner. 
<img width="181" alt="screen shot 2018-02-20 at 18 19 32" src="https://user-images.githubusercontent.com/9202899/36438580-a8338e26-166a-11e8-8d3f-2bbd29915096.png">


### todos

- [x] prop types

### testing

- Go to `https://calypso.localhost:3000` or [calypso.live](https://calypso.live/?branch=add/badge-component)
- Make sure the Badge (bottom right corner) is still present
- Any sections (e.g. branch name, ab tests) should still be there, depending on environment and configuration
